docs/resources/smn_message_template.md

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Simple Message Notification (SMN)"
+---
+
+# huaweicloud_smn_message_template
+
+Manages a SMN message template resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "name" {}
+variable "protocol" {}
+variable "content" {}
+
+resource "huaweicloud_smn_message_template" "test"{
+  name     = var.name
+  protocol = var.protocol
+  content  = var.content
```

Comment:
pls show the usage of parameter placeholders explicitly

Reply:
done